infra/scripts/migrate.sh

```
@@ -120,14 +123,14 @@ run_migration_cmd() {
     print_status "$description..."
     
     # Check if we're running in Docker environment
-    if docker compose -f ./docker/docker-compose.yml ps mysql | grep -q "Up"; then
+    if docker compose --env-file $ENV_FILE -f $DOCKER_COMPOSE_FILE ps mysql | grep -q "Up"; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)